### ISSUE TITLE
Allow STAB bonus to be customized

### ISSUE THREAD

**Issue by LilJess13** [Auroratide/poke5e#76]
Right now the STAB damage is just doubling the move power. My currently level 5 pokemon is getting +2 for STAB when it should only be getting +1 due to its level.

My recommendation: STAB should be its own individually adjustable stat for the pokemon that is then added to the damage.

**Comment by Auroratide** [comment]
Yeah, it was intentional. At the time my group felt the by-the-book STAB bonus wasn't impactful, so we doubled the stat modifier instead of deriving it from the level. It also made the rule easier to memorize prior to implementing STAB on the site.

I could get behind making STAB one of the advanced customization options though.

**Comment by LilJess13** [comment]
That's a fair reason for it to be the way it was. A custom option would be great for sure though. Customization is always best!
